policies/mandatory-file-License.yml

```diff
@@ -1,6 +1,6 @@
 # metadata
-name: This repo is missing a license file
-description: This is a config to check if LICENSE.MD is present in repo.
+name: This repo is missing a LICENSE file
```

Comment:
Will this break existing issues already opened?

Reply:
There will not be any problem to opened issues.